### PR TITLE
Seperate apps from onos-ran docker container

### PR DESCRIPTION
**Pull request by woojoong88**
Seperate apps from onos-ran docker container - for microservice.

### CODE REVIEW

**Comment by woojoong88** [comment]
I just make new Dockerfiles for each application in the same repo, not make new repos for each apps.
If necessary, I will do it, but looks like not now. Any opinion @SeanCondon ?

**Comment by adibrastegarnia** [comment]
@woojoong88 
Not sure we need to push apps images or not to the Dockerhub but that would be good to do that for onit that we can deploy them easily.

File: Makefile [hunk 1]
```diff
@@ -66,13 +68,23 @@ onos-ran-docker: onos-ran-base-docker # @HELP build onos-ran Docker image
...
 	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o build/onos-ran-tests/_output/bin/onos-ran-tests ./cmd/onos-ran-tests
 	docker build . -f build/onos-ran-tests/Dockerfile -t onosproject/onos-ran-tests:${ONOS_RAN_VERSION}
 
 
 images: # @HELP build all Docker images
-images: build onos-ran-docker onos-ran-tests-docker
+images: build onos-ran-docker onos-ran-ho-docker onos-ran-mlb-docker onos-ran-tests-docker
 
 kind: # @HELP build Docker images and add them to the currently configured kind cluster
 kind: images
```
Comment (SeanCondon) [inline Makefile]: there should be a `kind` entry for each one of `ho` and `mlb` too

Comment (woojoong88) [reply Makefile]: Oh, sorry I just saw this comment.. Let me change here later @SeanCondon .

Comment (woojoong88) [reply Makefile]: I did :D

**Pull request merged**